packages/@aws-cdk/aws-glue/lib/job.ts

```diff
@@ -0,0 +1,872 @@
+  }
+}
+
+/**
+ * Python version
+ */
+export enum PythonVersion {
+  /**
+   * Python 2 (the exact version depends on GlueVersion and JobCommand used)
+   */
+  TWO = '2',
+
+  /**
+   * Python 3 (the exact version depends on GlueVersion and JobCommand used)
+   */
+  THREE = '3',
+}
+
+/**
+ * Job states emitted by Glue to CloudWatch Events.
+ *
+ * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#glue-event-types for more information.
+ */
+export enum JobState {
+  /**
+   * State indicating job run succeeded
+   */
+  SUCCEEDED = 'SUCCEEDED',
+
+  /**
+   * State indicating job run failed
+   */
+  FAILED = 'FAILED',
+
+  /**
+   * State indicating job run timed out
+   */
+  TIMEOUT = 'TIMEOUT',
+
+  /**
+   * State indicating job is starting
+   */
+  STARTING = 'STARTING',
+
+  /**
+   * State indicating job is running
+   */
+  RUNNING = 'RUNNING',
+
+  /**
+   * State indicating job is stopping
+   */
+  STOPPING = 'STOPPING',
+
+  /**
+   * State indicating job stopped
+   */
+  STOPPED = 'STOPPED',
+}
+
+/**
+ * The Glue CloudWatch metric type.
+ *
+ * @see https://docs.aws.amazon.com/glue/latest/dg/monitoring-awsglue-with-cloudwatch-metrics.html
+ */
+export enum MetricType {
+  /**
+   * A value at a point in time.
+   */
+  GAUGE = 'gauge',
+
+  /**
+   * An aggregate number.
+   */
+  COUNT = 'count',
+}
+
+/**
+ * The job command name used for job run.
+ *
+ * If you need to use a JobCommandName that doesn't exist as a static member, you
+ * can instantiate a `JobCommandName` object, e.g: `JobCommandName.of('other name')`.
+ */
+export class JobCommandName {
+  /**
+   * Command for running a Glue ETL job.
+   */
+  public static readonly ETL = new JobCommandName('glueetl');
+
+  /**
+   * Command for running a Glue streaming job.
+   */
+  public static readonly STREAMING = new JobCommandName('gluestreaming');
+
+  /**
+   * Command for running a Glue python shell job.
+   */
+  public static readonly PYTHON_SHELL = new JobCommandName('pythonshell');
+
+  /**
+   * Custom command name
+   * @param name command name
+   */
+  public static of(name: string): WorkerType {
+    return new JobCommandName(name);
+  }
+
+  /**
+   * The name of this JobCommandName, as expected by Job resource.
+   */
+  public readonly name: string;
+
+  private constructor(name: string) {
+    this.name = name;
+  }
+
+  /**
+   * The worker type name as expected by Job resource.
+   */
+  public toString(): string {
+    return this.name;
+  }
+}
+
+/**
+ * JobCommand specifies the execution environment and the code executed when a job is run.
+ */
+export class JobCommand {
+
+  /**
+   * Create a glueetl JobCommand with the given scriptLocation
+   *
+   * @param scriptLocation specifies the Amazon Simple Storage Service (Amazon S3) path to a script that executes a job.
+   * @param pythonVersion specifies the Python shell version for the ETL job. Versions supported vary depending on GlueVersion.
```

Comment:
what is the default value for pythonVersion if it's not supplied?

Reply:
I can't complain enough abt the lack of this type of info in documentation
- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-job-jobcommand.html
- https://docs.aws.amazon.com/glue/latest/webapi/API_JobCommand.html

My team uses scala, so we're not using Python attributes at all.

This is a screenshot from one of our existing scala jobs
<img width="580" alt="glue_existing_scala_job" src="https://user-images.githubusercontent.com/5001/127637342-d5ab3dc1-112c-4cb5-a660-0db2e0325b80.png">

This is a screenshot from attempting to create a new glue job via console
<img width="656" alt="glue_new_job" src="https://user-images.githubusercontent.com/5001/127637344-59e53f98-1312-4223-ae0f-d436f9fc2a1d.png">

From the above, I think Python 2 is the default.